packages/twenty-server/src/workspace/workspace-query-runner/utils/compute-pg-graphql-error.util.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  BadRequestException,
+  HttpException,
+  InternalServerErrorException,
+} from '@nestjs/common';
+
+interface PgGraphQLErrorMapping {
+  [key: string]: (command: string, objectName: string) => HttpException;
+}
+
+const pgGraphQLErrorMapping: PgGraphQLErrorMapping = {
+  'delete impacts too many records': (command, objectName) =>
+    new BadRequestException(
+      `Cannot ${command} ${objectName} because it impacts too many records (check MAXIMUM_RECORD_AFFECTED environment variable)`,
```

Comment:
Considering this error is client side and user-facing (BadRequestException), I wouldn't specify "(check MAXIMUM_RECORD_AFFECTED environment variable)" 🤔

Reply:
I'll remove it :)